src/modules/did/did.js

```diff
@@ -730,29 +682,35 @@ class DIDModule {
    * Throws NoDID if the DID does not exist on chain.
    * @param {string} did - The DID can be passed as fully qualified DID like `did:dock:<SS58 string>` or
    * a 32 byte hex string
+   * @param getOffchainSigKeys
    * @return {Promise<object>} The DID document.
    */
   // eslint-disable-next-line sonarjs/cognitive-complexity
   async getDocument(did, { getOffchainSigKeys = true } = {}) {
```

Comment:
Can you please update this method to use `didDetails` RPC so avoid multiple calls to the chain?

Reply:
This should be done separately.